pkg/apis/core/types.go

```diff
@@ -5317,6 +5317,12 @@ type WindowsSecurityContextOptions struct {
 	// PodSecurityContext, the value specified in SecurityContext takes precedence.
 	// +optional
 	RunAsUserName *string
+
+	// HostProcess determines if a container should be run as a 'Host Process' container.
+	// This field is alpha-level and will only be honored by components that enable the
```

Comment:
I will update this comment to reflect how we will fail validation if flag is used by future is not set (like below)